### PR TITLE
Use Trip Name as Notification Title [OTP-1184]

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [na] The description lists all applicable issues this PR seeks to resolve
- [na] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

This PR changes the notification title to the trip name for trip monitoring push notifications.

### CODE REVIEW

File: src/main/java/org/opentripplanner/middleware/utils/NotificationUtils.java [hunk 1]
```diff
@@ -54,21 +55,23 @@ public class NotificationUtils {
      * See https://support.twilio.com/hc/en-us/articles/360033806753-Maximum-Message-Length-with-Twilio-Programmable-Messaging
      */
     private static final int SMS_MAX_LENGTH = 320;
-    /** Lowest permitted push message length between Android (240) and iOS (178). */
-    private static final int PUSH_MESSAGE_MAX_LENGTH = 178;
+    /** Lowest permitted length (title and message) btw Android (240 for message) and iOS (178 for title + message). */
+    public static final int PUSH_TOTAL_MAX_LENGTH = 178;
+    /** Lowest permitted push title length between Android (65) and iOS (none). */
```
Comment (amy-corson-ibigroup) [inline src/main/java/org/opentripplanner/middleware/utils/NotificationUtils.java]: I think the comments are a little misleading here, using "lowest" makes me think we're talking about the minimum number of characters that can be in a push notification, but judging by the variable I think we're probably talking about maximum?

File: src/test/java/org/opentripplanner/middleware/utils/NotificationUtilsTestCI.java [hunk 2]
```diff
@@ -61,6 +61,7 @@ public void canSendPushNotification() {
             // Conventionally user.email
             push,
             "Tough little ship!",
```
Comment (amy-corson-ibigroup) [inline src/test/java/org/opentripplanner/middleware/utils/NotificationUtilsTestCI.java:63]: :)

**Review by amy-corson-ibigroup — APPROVED** [review]
Looks good to me!

File: src/main/java/org/opentripplanner/middleware/utils/NotificationUtils.java (hunk 1 above)
Comment (binh-dam-ibigroup) [reply src/main/java/org/opentripplanner/middleware/utils/NotificationUtils.java]: I rephrased in https://github.com/ibi-group/otp-middleware/pull/229/commits/579b836885b076a8e69a23f22d4ac1c30fe5ee26.

File: src/test/java/org/opentripplanner/middleware/utils/NotificationUtilsTestCI.java (hunk 2 above)
Comment (JymDyerIBI) [reply src/test/java/org/opentripplanner/middleware/utils/NotificationUtilsTestCI.java:63]: Phil liked this notification as well.

**Review by JymDyerIBI — APPROVED** [review]
👍  Looks good to me!

**Pull request merged**